public/javascripts/languages.js

```diff
@@ -0,0 +1 @@
+var LANGUAGE_OPTIONS = ['english', 'mandarin', 'spanish', 'portuguese', 'bahasa']
```

Comment:
load these from a database. In the admin view, let the admin add/remove languages. Include language codes (based on ISO https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)